website/docs/r/site_recovery_replication_recovery_plan.html.markdown

```diff
@@ -258,6 +261,8 @@ A `recovery_group` block supports the following:
 
 * `type` - (Required) The Recovery Plan Group Type. Possible values are `Boot`, `Failover` and `Shutdown`.
 
+**Note:** The List of `recovery_group` must be in the order of `Shutdown`-`Failover`-`Boot`.
```

Comment:
should these be modelled differently to account for that?

Reply:
have updated per comments, please take another look:)